app/util/jira/populate_db.sh

```diff
@@ -231,13 +245,39 @@ else
   exit 1
 fi
 
-echo "Step8: Start Jira"
+echo "Step9: Update jira license in database"
+if [[ -s ${JIRA_LICENSE_FILE} ]]; then
+  LICENSE=$(cat ${JIRA_LICENSE_FILE})
+  LICENSE_ID=$(PGPASSWORD=${JIRA_DB_PASS} psql -h ${DB_HOST} -d ${JIRA_DB_NAME} -U ${JIRA_DB_USER} -Atc \
+  "select id from productlicense;")
+  if [[ -z "${LICENSE_ID}" ]]; then
+    echo "The LICENSE_ID variable is empty. Please check that the license is exist in the database."
```

Comment:
Let's change the message to:
`echo "License update failed. License id value in the database is empty."`

Reply:
replace